cmd/cloudshell_open/project.go

```diff
@@ -43,8 +43,12 @@ func listProjects() ([]string, error) {
 	}
 	b = bytes.TrimSpace(b)
 	p := strings.Split(string(b), "\n")
-	sort.Strings(p)
-	return p, err
+	if len(p) == 1 && p[0] == "" {
+		return []string{}, err
+	} else {
```

Comment:
no need for `else{` because you return in previous block.

Reply:
Hmm...  Don't I need to return if the if didn't match?

Reply:
I meant this, if you do

```
if cond {
   return
} else {
   logic
   return
}
```

several Go linting tools would tell you `else is superfluous` as it's equivalent to

```
if cond {
   return
}

logic
return
```

which is more readable.

Reply:
Ah. I see.  I'm used to expression-oriented ifs :)